Tests/FunctionalTests/ClangModuleTests.swift

```diff
@@ -137,6 +137,34 @@ class ClangModulesTestCase: XCTestCase {
             XCTAssertFileExists(debugPath.appending(component: "CDynamicLookup".soname))
         }
     }
+    
+    func testObjectiveCPackageWithTestTarget(){
+#if os(macOS)
+        fixture(name: "ClangModules/ObjCmacOSPackage") { prefix in
+            // Build the package.
+            XCTAssertBuilds(prefix)
+            XCTAssertFileExists(prefix.appending(components: ".build", "debug", "ObjCmacOSPackage".soname))
+            // Run swift-test on package.
+            XCTAssertSwiftTest(prefix)
```

Comment:
We only need the test till this point, you can remove the XCTestHelper test (we already have test for that iirc)